libs/design/src/molecules/list/list/list.component.spec.ts

```diff
@@ -109,14 +109,4 @@ describe('DaffListComponent', () => {
       });
     });
   });
-
-  describe('<daff-nav-list>', () => {
```

Comment:
why remove this test entirely?

Reply:
It's a duplicate test.

Reply:
https://www.shopgoodwill.com/Item/117857028

Reply:
It's a duplicate test. It's already tested in line 67.